ProactivePolicyTemplate/build.cake

```diff
@@ -314,6 +314,6 @@ RunTarget(target);
 //////////////////////////////////////////////////////////////////////
 
 string ToolsExePath(string exeFileName) {
-    var exePath = System.IO.Directory.GetFiles(@".\Tools", exeFileName, SearchOption.AllDirectories).FirstOrDefault();
+    var exePath = System.IO.Directory.GetFiles(@"./Tools", exeFileName, SearchOption.AllDirectories).FirstOrDefault();
```

Comment:
This should be OS independent, like `Path.Combine`. Most probably it will fail for windows.

Reply:
Double checked, Polly uses same slash https://github.com/App-vNext/Polly/blob/master/build.cake#L305